guides/common/modules/proc_using-ansible-vault-with-project.adoc

```diff
@@ -0,0 +1,39 @@
+[id="using-ansible-vault-with-{project-context}_{context}"]
+= Using Ansible Vault with {Project}
+
+You can encrypt sensitive Ansible data files using the Ansible Vault tool and configure Ansible to access the encrypted files using a password stored in a file.
+
+.Procedure
+. If you customized `/etc/ansible/ansible.cfg`, copy your configuration from `/etc/ansible/ansible.cfg` to `/usr/share/foreman-proxy/.ansible.cfg`.
+. Encrypt the sensitive file using the `ansible-vault` command:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# ansible-vault encrypt /etc/ansible/roles/_Role_Name_/vars/main.yml
+----
++
+Note that `ansible-vault` changes the file permissions to `600`.
+. Change the group and permissions of the encrypted file to ensure that the `foreman-proxy` user can read it:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# chown :foreman-proxy /etc/ansible/roles/_Role_Name_/vars/main.yml
```

Comment:
Definitely a non-blocking comment, just I am not aware of this writing convention here ,hence giving this suggestion :) , rest looks good as I see @nofaralfasi has already tested.
```suggestion
# chown :foreman-proxy /etc/ansible/roles/_role_name_/vars/main.yml
```

Reply:
It's mostly `_My_Role_Name_` so that "My_Role_Name" is shown in italics.